CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+ [Added]
+ - Possibility to show a warning in disco-tpl from a config file
```

Comment:
Change to 'Possibility to show a warning in disco-tpl' instead of 'Possibility to show a warning in disco-tpl from a config file'